tests/functional/modules/test_zos_copy_func.py

```diff
@@ -13,13 +13,15 @@
 
 from __future__ import absolute_import, division, print_function
 
+from ibm_zos_core.tests.helpers.users import ManagedUserType
 import pytest
 import os
 import shutil
 import re
 import time
 import tempfile
 import subprocess
+from ibm_zos_core.tests.helpers.users import *
```

Comment:
I will discourage the use of `import *`, it obfuscates the function or class origin and makes more difficult to find namespace collision errors.

Reply:
Done, appended `from ibm_zos_core.tests.helpers.users import ManagedUserType, ManagedUser`, VSCode substituted the `*`.